practice/react-practice-one/src/components/Pagination/Pagination.jsx

```diff
@@ -0,0 +1,99 @@
+/* Import dependencies */
+import PropTypes from 'prop-types';
+
+/* Import components */
+import Button from '../Buttons/Button';
+
+/* Import CSS */
+import './pagination.css';
+
+const Pagination = ({ currentPage, totalRecords, pageLimit, onPageChange }) => {
+  const totalPages = Math.ceil(totalRecords / pageLimit);
+
+  const handleClick = (page) => {
+    if (page !== currentPage && page >= 1 && page <= totalPages) {
+      onPageChange(page);
+    }
+  };
+
+  const getPageNumbers = () => {
+    const pages = [];
+    const ellipsis = '...';
+
+    if (totalPages <= 5) {
```

Comment:
@JennyDoJD can we reduce the if else statement?

Reply:
Maybe it's okay. I will refactor it.

Reply:
I refactored